### PR TITLE
DR-1114 add perf test action

### PR DESCRIPTION
**Pull request by snf2ye**
Github actions changes: https://github.com/broadinstitute/datarepo-actions/pull/23

### CODE REVIEW

File: datarepo-clienttests/src/main/resources/logback.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{50} - %msg%n</pattern>
         </encoder>
     </appender>
-    <root level="INFO">
+    <root level="DEBUG">
```
Comment (snf2ye) [inline datarepo-clienttests/src/main/resources/logback.xml]: TODO: change back before merging

File: .github/workflows/gradle-build-pr.yml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 name: Integration test and Connected tests
+env:
+  K8_CLUSTER: "integration-master"
+  GOOGLE_APPLICATION_CREDENTIALS: '/tmp/jade-dev-account.json'
```
Comment (smark88) [inline .github/workflows/gradle-build-pr.yml:4]:
```suggestion
  GOOGLE_APPLICATION_CREDENTIALS: '${GOOGLE_APPLICATION_CREDENTIALS}'
```

File: .github/workflows/gradle-build-pr.yml [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+  GOOGLE_SA_CERT: '/tmp/jade-dev-account.pem'
```
Comment (smark88) [inline .github/workflows/gradle-build-pr.yml:5]:
```suggestion
  GOOGLE_SA_CERT: '${GOOGLE_SA_CERT}'
```

**Comment by snf2ye** [comment]
moved to new PR

**Pull request closed without merging**